CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Grafana Mimir
 
+* [FEATURE] Add capability to configure the S3 storage class. #3438
```

Comment:
```suggestion
* [FEATURE] Added `-<prefix>.s3.storage-class` flag to configure the S3 storage class for objects written to S3 buckets. #3438
```

Reply:
Hope you don't mind @inbarpatashnik, going to commit this minor edit so we can get this merged :)